### PR TITLE
Can configure different middleware for each service

### PR DESCRIPTION
**Pull request by sunfmin**

### CODE REVIEW

File: harness.go [hunk 1]
```diff
@@ -31,12 +33,41 @@ func (s search) Description() grpc.ServiceDesc {
...
+
 func main() {
+
+	mux := http.NewServeMux()
+	a := account{}
 	s := search{}
 
-	mux := wrapper.Wrap(s)
+	wrapper.Handle(mux, a, mustLogin(wrapper.Wrap(a)))
+	wrapper.Handle(mux, s, wrapper.Wrap(s))
```
Comment (bodhi) [inline harness.go]: I find it a bit awkward that you need to pass `s` and `wrapper.Wrap(s)` to `Handle`.

Comment (sunfmin) [inline harness.go]: because sometime you need to wrap `wrapper.Wrap(s)`...

Comment (bodhi) [inline harness.go]: Then pass the middleware directly?

```diff
-	wrapper.Handle(mux, a, mustLogin(wrapper.Wrap(a)))
+	wrapper.Handle(mux, a, mustLogin)
```

Comment (sunfmin) [inline harness.go]: What If they want to wrap multiple middleware?

**Pull request merged**